frontend/packages/kubevirt-plugin/src/k8s/patches/vm/vm-cdrom-patches.ts

```diff
@@ -99,6 +99,7 @@ export const getCDsPatch = (
         const storageClassName = dataVolumeWrapper.getStorageClassName();
```

Comment:
we should use it here as well. 
Another problem is still present here: the CDRom modal should not allow editing of URL cdrom which were already created.

Should we create another bug for that or fix it here?

Reply:
I added the autogeneration to the DV name. I think we should handle the other problem as a separate bug at this point.

Reply:
yes, created the bug already